tools/cell_census_builder/consolidate.py

```diff
@@ -17,12 +17,20 @@ def consolidate(args: argparse.Namespace, uri: str) -> None:
         return
 
     logging.info("Consolidate: started")
+    uris_to_consolidate = gather(uri)
+    run(args, uris_to_consolidate)
+    logging.info("Consolidate: finished")
+
 
+def gather(uri: str) -> List[str]:
```

Comment:
if these are module-private functions, please add an underscore prefix (e.g., _gather).  If public functions, please give them a more descriptive name.

Ditto for run/_run